api/tests/court-cases.integration.test.ts

```diff
@@ -15,16 +15,34 @@ describe("/court-cases", () => {
   })
 
   describe("GET", () => {
+    it("returns a 401 status code if authorization header is not set", async () => {
+      const response = await request(app)
+        .get("/court-cases")
+        .query(stringify({ forces: ["01"], maxPageItems: "10" }))
```

Comment:
Is it worth putting `stringify({ forces: ["01"], maxPageItems: "10" })` in a variable? I've seen it used a couple of times.

Reply:
I thought about it, but wasn't sure if it would affect the readability of the tests.

Reply:
Fair